### DISCUSSION TITLE
How to work with  vue 2.7

### DISCUSSION THREAD

**Discussion by fyeeme** [vuejs/pinia#1380] [Help and Questions] [answered]
as far as I know vue 2.7 no longer need @vue/composition-api, and new npm module `@vitejs/plugin-vue2` show the demo that vite works with vue 2.7. 
@vitejs/plugin-vue2's provide playground main.ts code
```ts 
import Vue from 'vue'
import App from './App.vue'

new Vue({
  render: h => h(App),
}).$mount('#app')
```

my  question is pinia how to intergration with vue 2.7

**Comment by posva** [comment, marked as answer]
You don't need to change anything in pinia to make it work with 2.7

**Reply by fyeeme** [reply]
Yes, pinia's doc has make it much more clear  to work with vue 2, my mistake

**Reply by jonatan-mobal** [reply]
@posva Can we remove @vue/composition-api when using Pinia with Vue 2.7?

**Reply by posva** [reply]
yes, you should. Make sure to upgrade your other dependencies too. Especially `vue-demi`.

**Reply by jonatan-mobal** [reply]
That's great, thank you!
